DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Helpers/EmbeddedContentItemsGraphSyncer.cs

```diff
@@ -456,5 +456,24 @@ private ContentItem[] ConvertToContentItems(JArray? contentItems)
                 throw new GraphSyncException("Embedded content container does not contain ContentItems.");
             return embeddedContentItems;
         }
+
+        public async Task AddRelationship(IDescribeRelationshipsContext context)
```

Comment:
This method looks as if it isn't called.
I'd expect it to be called by the part syncers that have embedded content items (flows, taxonomies & bags).